develop/tutorials/articles/110-portlets/04-mvc-resourcecommand.markdown

```diff
@@ -47,7 +47,7 @@ property to `MVCResourceCommand.class`:
     }
 
 As a real-world example, consider the Login Portlet's 
-[`CaptchaMVCResourceCommand` class](https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/login/login-web/src/main/java/com/liferay/login/web/portlet/action/CaptchaMVCResourceCommand.java): 
+[`CaptchaMVCResourceCommand` class](https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/CaptchaMVCResourceCommand.java): 
```

Comment:
Thanks for pointing this out. I took the opportunity to reference the source code instead of linking to github. You can follow the PR here, which includes some of the other link fixes you specified as well: https://github.com/sez11a/liferay-docs/pull/1866